### PR TITLE
Fixed additional additional CTC function bug

### PR DESCRIPTION
**Pull request by Amy-Xu**
1) _iitax and _fica didn't sum up to combined tax liabilities. This problem should be fixed in this PR

2) Also added phaseout to this credit

@MattHJensen Could you review? Thanks!

### CODE REVIEW

File: taxcalc/current_law_policy.json [hunk 1]
```diff
@@ -1123,7 +1123,36 @@
...
+        "row_label": ["2013", "2014", "2015"],
+        "cpi_inflated": false,
+        "col_label": "",
+        "value":     [[0, 0, 0, 0],
+		      [0, 0, 0, 0],
+		      [150000, 300000, 150000, 200000]]     
+    },
+    "_CTC_additional_prt":{
+        "long_name": "Additional additional child tax credit amount phaseout rate (Lee-Rubio)",
+	"description": "The amount of additional additional child tax credit is reduced at this rate per dollor exceeding the phaseout starting AGI. ",
```
Comment (talumbau) [inline taxcalc/current_law_policy.json]: `dollor` -> `dollar`

**Comment by MattHJensen** [comment]
@Amy-Xu, this looks good to me once the tests are passing.

File: taxcalc/functions.py [hunk 2]
```diff
@@ -1639,11 +1639,14 @@ def DEITC(c59660, c07100, c08800, c05800, _avail, e11601, e07170, _othertax):
 
 @iterate_jit(nopython=True)
 def IITAX(c09200, c59660, c11070, c10960, c11600, c10950, _eitc, e11580,
-          e11450, e11500, e82040,
-          e11550, e10000, _fica, _personal_credit, n24, CTC_additional):
+          e11450, e11500, e82040, e09900, e11400, e11300, e11200, e11100,
+          e11550, e09710, e09720, e10000, _fica, _personal_credit, n24,
+          CTC_additional, CTC_additional_ps, CTC_additional_prt, c00100,
+          _sep, MARS):
```
Comment (talumbau) [inline taxcalc/functions.py]: Hmm.. I think the failed test is due to the additional function arguments here in `IITAX`. It is curious that these arguments are here but there is no change in the function implementation. @Amy-Xu is that right?

Comment (MattHJensen) [inline taxcalc/functions.py]: That looks right to me. @talumbau, I am about to checkout Amy's branch and then open a new PR that fixes this.

Comment (talumbau) [inline taxcalc/functions.py]: hmm.. what about `e09900`? Martin's test will fail on added function arguments that are not used in the implementation of the function.

Comment (MattHJensen) [inline taxcalc/functions.py]: I found a few that we unused. You'll see the ones I deleted in a minute when I open the PR. Could you check to make sure that I only deleted unused arguments?

**Comment by Amy-Xu** [comment]
@talumbau @MattHJensen Thanks for the comments. I'll test the other PR. Closing.

**Pull request closed without merging**